app/models/manageiq/providers/google/inventory/parser.rb

```diff
@@ -716,4 +720,16 @@ def parse_uid_from_url(url)
     # returns the last component of the url
     url.split('/')[-1]
   end
+
+  def deprecated_image?(image)
+    image.kind == "compute#image" ? !image.deprecated.nil? : false
+  end
+
+  def skip_image?(image)
+    return false if options.get_deprecated_images
+    return false unless deprecated_image?(image)
+    return false if @active_images.include?(image.id.to_s)
```

Comment:
Is this exercised?

`images` calls `skip_image?` before calling `image`, but `image` is what sets any values in `@active_images`

Reply:
Yes, there are a number of deprecated images that are in use in the current VCR.  And `#image` isn't what sets `@active_images`, `#instances` is [[ref]](https://github.com/ManageIQ/manageiq-providers-google/pull/199/files#diff-58b4cfe58ce26bfb4174844bdcaa11ee53d8a541cadb253d70f7c3d9c0c92557R191) which is called before `#images` [[ref]](https://github.com/ManageIQ/manageiq-providers-google/pull/199/files#diff-58b4cfe58ce26bfb4174844bdcaa11ee53d8a541cadb253d70f7c3d9c0c92557L38-R42)

Reply:
(while running the refresher spec)
```
   728:   def skip_image?(image)
   729:     return false if options.get_deprecated_images
   730:     return false if !deprecated_image?(image)
   731:     byebug if @active_images.include?(image.id.to_s)
=> 732:     return false if @active_images.include?(image.id.to_s)
   733: 
   734:     true
   735:   end
   736: end
(byebug) deprecated_image?(image)
true
(byebug) @active_images.include?(image.id.to_s)
true
```

Reply:
I really don't understand how this could work :/  Can we review via video?

Reply:
Added a comment re: ordering of `instances` vs `images` being called above

Reply:
Thanks @agrare  - I had misread the diff and didn't realize that instances was before images (I just saw a list and assumed it was alphabetical).  That comment helps a lot.